pom.xml

```diff
@@ -119,10 +120,106 @@
             <version>${lucene.version}</version>
             <scope>test</scope>
         </dependency>
+        <!--Require newer version than inherited-->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>27.0.1-jre</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-runtime</artifactId>
+            <version>${kafka.version}</version>
+            <classifier>test</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-runtime</artifactId>
+            <version>${kafka.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>${kafka.version}</version>
+            <classifier>test</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <version>${kafka.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka_2.11</artifactId>
+            <type>test-jar</type>
+            <classifier>test</classifier>
+            <scope>test</scope>
+            <version>${kafka.version}</version>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${version.docker.maven.plugin}</version>
```

Comment:
I'm not convinced that the profile strategy is the best strategy here. By default, we'd want to run integration tests against all versions, and we'd want a common jenkins command to work for all connectors -- meaning ideally `mvn verify` (or `install`) would run against both es5 and es6 images. This is how Debezium project also works: all containers are brought up by default claiming different ports, and all tests run by default. Since maven already gives control over which integration test to run, I think giving our devs the granular control over which docker images to bring up and down is probably a premature optimization that adds more complexity to an already complex pom.

WDYT?

Reply:
Okay, that sounds good. I think for JDBC we'll want to allow limiting to testing against specific DBMSes, and profiles may work there. But I agree, by default we probably want to run against all of them.